README.md

```diff
@@ -1 +1,102 @@
 # awesome-selfhosted-data
+
+**Status: Draft**
+
+This repository holds data used to generate https://github.com/awesome-selfhosted/awesome-selfhosted
+
+## Contributing
+
+Don't know where to start? Check issues labeled [`help wanted`](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22), [`fix`](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues?q=is%3Aissue+is%3Aopen+label%3Afix) and [`curation`](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues?q=is%3Aissue+is%3Aopen+label%3Acuration).
+
+
+### Maintenance
+
+- Software with no development activity for 6-12 months may be removed from the list
+- Unmaintained software without an active community and/or persistent security issues may be removed from the list
+- Problems should be reported automatically: [![](https://img.shields.io/travis/awesome-selfhosted/awesome-selfhosted/master?label=link%20checks)](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/1)
+
+
+### Add new software to the list
+
+- [ ] Please submit one item per issue/pull request. This eases reviewing and speeds up the addition process.
+- [ ] Please check that your addition is not already listed at any of these lists (If it fits in one of those lists, please try to get it added there instead):
+  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin)
+  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics)
+  - [staticgen.com](https://www.staticgen.com/)
+  - [staticsitegenerators.net](https://staticsitegenerators.net/)
+- [ ] Please search for relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [pull requests](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
+- [ ] [Create a new `.yml` file](https://github.com/nodiscc/awesome-selfhosted-data/new/master/software) based on the template below in the `software/` directory:
+
+```yaml
+# projects/my-awesome-software.yml
+name: "My awesome software" # required
+website_url: "https://my.awesome.softwar.e" # required
+source_code_url: "https://gitlab.com/awesome/software" # required if different from website URL
```

Comment:
source_code_url should be required and website_url should be optional.

Reply:
you are right, this rule would be simpler. However it requires a bit more work: the import script (https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/3) does not know whether `- [Software name](https://software.na.me)...` from the original list is the `website_url` or the `source code_url`. So it is always imported as `website_url`. I see two ways around this:
- Modify the import script so that it populates `source_code_url` instead of `website_url` when only the main `- [Software name](https://software.na.me)...` link is present
- Apply this change manually on the imported data in https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/3 (for all `software/*.yml` that does not contain `source_code_url`, replace `website_url` with `source_code_url` (probably easier - or not).

In any case https://github.com/nodiscc/hecat/  needs to be modified to use `source_code_url` as main URL when no `website_url` is present (currently `website_url` is a hard requirement).

Looking into it

Reply:
Modifying the import script should be the easier option and shouldnt be too hard to implement.

Reply:
Modifying the import script should be the easier option and shouldnt be too hard to implement.

Reply:
in progress at https://github.com/nodiscc/hecat/pull/4

Reply:
https://github.com/nodiscc/hecat/pull/4 completed, https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/2/commits/02de41d1f5490b80338938141942ab887cd5e88c